test/functional/utils/assertion-helper.js

```diff
@@ -0,0 +1,11 @@
+var expect = require('chai').expect;
+
+
+module.exports = {
+    // NOTE: this method checks, is the specified message the part of the value of keys given in testError object.
+    checkTestErrorContainsMessage: function (testError, message) {
+        Object.keys(testError).forEach(function (key) {
+            expect(testError[key]).with.deep.property('[0]').that.contains(message);

```

Comment:
Also, what if each browser has multiple errors? How can I check second error? We need to pass index as parameter here